applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -30,6 +30,9 @@ return view.extend({
 				"point at the router's public IP address. " +
 				"Once configured, issuing certificates can take a while. " +
 				"Check the logs for progress and any errors.") + '<br/>' +
+				_('You can sign certificates for free with:') + '<br/>' +
+				'<a href="https://ZeroSSL.com/" target="_blank">ZeroSSL</a> ' + _('Default') + '<br/>' +
+				'<a href="https://LetsEncrypt.org/" target="_blank">LetsEncrypt</a> ' + '<br/>' +
```

Comment:
We override the upstream default in the acme.sh wrapper to use letsencrypt by default. The upstream change to zerossl was for commercial reasons (they are sponsoring the acme.sh project), and I personally consider it pretty dodgy. So please don't mention them here. Instead, we should write something like "letsencrypt will be used by default, but a custom ACME server can be used by specifying the `acme_server` setting".

Reply:
Also, the wiki page you're linking to below should probably be updated to reflect this as well :)

Reply:
Well, the idea is that if you set a custom value for the ACME server you know what you're doing and so that takes precedence over the value of `staging`. I think this is fine; but we can certainly add a warning if someone sets a custom value and also enables `staging`; but in terms of UI, I think the "custom server" field should just be hidden as an "advanced option" somewhere...